src/peers/PeersTable/PeersTable.js

```diff
@@ -2,16 +2,33 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'redux-bundler-react'
 import { translate, Trans } from 'react-i18next'
-import { Table, Column, AutoSizer } from 'react-virtualized'
+import { Table, Column, AutoSizer, SortDirection } from 'react-virtualized'
 import CountryFlag from 'react-country-flag'
 import Cid from '../../components/cid/Cid'
 
+function sortByProperty (arr, property, direction) {
+  if (!arr) return arr
+  const dirModifier = direction === SortDirection.ASC ? 1 : -1
+  return arr.sort(({ [property]: a }, { [property]: b }) => (a == null) - (b == null) || dirModifier * +(a > b) || dirModifier * -(a < b))
```

Comment:
so fancy. We should probably extract this to a seperate file we can use else where and test.

Reply:
Happy to make that happen. I'm thinking `/src/lib/sort.js` seems like a good home.